src/cli/domain/get-components-deps.js

```diff
@@ -5,28 +5,41 @@ var path = require('path');
 var _ =  require('underscore');
 
 module.exports = function(components){
+  var deps = { modules: {}, withVersions: {}, templates: {} };
 
-  var deps = { modules: [], withVersions: [] };
-
-  _.forEach(components, function(c){
+  var legacyTemplates = {
+    'jade': true,
+    'handlebars': true
+  };
 
+  components.forEach(function(c){
     var pkg = fs.readJsonSync(path.join(c, 'package.json'));
+    var type = pkg.oc.files.template.type;
+    var dependencies = pkg.dependencies;
 
-    _.forEach(_.keys(pkg.dependencies), function(d){
+    if (!deps.templates[type] && !legacyTemplates[type]) {
```

Comment:
That's why I put it as question. I don't have strong opinions on this. Let's leave it as is if you like.